autoload/gitgutter/hunk.vim

```diff
@@ -422,7 +422,7 @@ function! s:open_hunk_preview_window()
             \ })
 
       call setbufvar(winbufnr(s:winid), '&filetype', 'diff')
-      call win_execute(s:winid, 'syntax enable')
+      call win_execute(s:winid, 'doautocmd FileType')
```

Comment:
`win_execute` might not be needed altogether: https://github.com/vim/vim/issues/4961